### PR TITLE
Add method for handling storage DB to CC

### PR DESCRIPTION
**Pull request by mdonaka**
# Summary
Add new DB to CC

# Purpose
Delete couchbase

# Contents
- Add storage DB
  - small stage: bind
  - dev stage: volumes
- Add method for new DB on CC

# Testing Methods Performed
- make test t=ComputationContainer p=small
- temporarily switch DB and request from libClient
- CI

# Step
1. Add method on MC
2. **[now]** Add method on CC
3. Switch DB and test with libClient
4. Delete old DB

### CODE REVIEW

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+// shareの取り出し
+// qmpc::ComputationToDbGate::ValueTable Client::readShare(const std::string &data_id);
+TEST(ComputationToDbTest, SuccessReadShareTest)
+{
+    initialize();
+
+    const std::string data_id = "ReadShareTestId";
+    const std::string data =
+        "{\"value\":[[\"1\",\"2\"],[\"3\",\"4\"]]"
+        ",\"meta\":{\"piece_id\":0,\"schema\":[\"attr1\",\"attr2\"]}}";
```
Comment (arukuka) [inline src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp:31]: Use raw string literal if you don't mind.

-  https://cpprefjp.github.io/lang/cpp11/raw_string_literals.html
- https://en.cppreference.com/w/cpp/language/string_literal

**Review by arukuka — CHANGES_REQUESTED** [review]
Thank you for your contribution!  
I commented on the part that I was curious about, so please check it.

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+{
+    for (const auto &entry : fs::directory_iterator("/Db/share/"))
+    {
+        if (fs::is_directory(entry.path())) fs::remove_all(entry.path());
+    }
+    for (const auto &entry : fs::directory_iterator("/Db/result/"))
+    {
+        if (fs::is_directory(entry.path())) fs::remove_all(entry.path());
+    }
+}
```
Comment (come2ry) [inline src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp:20]: @mdonaka 
本番用の空間に影響しないように初期化する実装に変更お願いします！

**Review by come2ry — CHANGES_REQUESTED** [review]

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp (hunk 1 above)
Comment (mdonaka) [reply src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp]: @arukuka 
I didn't know that, thanks!
I fixed it in this 0ef7430e1492ac6e03e208db7e468281602b7179.

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp (hunk 2 above)
Comment (mdonaka) [reply src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp]: @come2ry 
You are right, thanks! I fixed in a79a00c35e94e1e3ea112aa4ce9472c0380c5649.

**Review by arukuka — APPROVED** [review]
Thank you!

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp (hunk 1 above)
Comment (arukuka) [reply src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp]: @mdonaka Thank you!  
I checked it.

File: src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp (hunk 2 above)
Comment (come2ry) [reply src/ComputationContainer/Test/UnitTest/CtoDBTest.cpp]: @mdonaka 
@arukuka 

LGTM!

**Comment by arukuka** [comment]
@mdonaka I beg your pardon, #50 was merged, please rebase this pr.

**Comment by mdonaka** [comment]
@arukuka 
Thank you! I rebased it.
I'll merge it when CI passes.

**Pull request merged**